Foundation/FileManager+Win32.swift

```diff
@@ -405,30 +405,36 @@ extension FileManager {
         guard alreadyConfirmed || shouldRemoveItemAtPath(path, isURL: isURL) else {
             return
         }
-        let faAttributes = try windowsFileAttributes(atPath: path)
+        let url = URL(fileURLWithPath: path)
+        var fsrBuf: [WCHAR] = Array<WCHAR>(repeating: 0, count: Int(MAX_PATH))
+        _CFURLGetWideFileSystemRepresentation(url._cfObject, false, &fsrBuf, Int(MAX_PATH))
+        let length = fsrBuf.firstIndex(where: {$0 == 0}) ?? (path.length * 2)
```

Comment:
This is probably better written as:

`let length = wcslen_s(&path, fsrBuf.count)`